src/wp-includes/class-wp-theme-json.php

```diff
@@ -1005,6 +1006,31 @@ public function get_stylesheet( $types = array( 'variables', 'styles', 'presets'
 		return $stylesheet;
 	}
 
+	/**
+	 * Returns the global styles custom css.
+	 *
+	 * @since 6.2.0
+	 *
+	 * @return string
```

Comment:
```suggestion
	 * @return string The global styles custom CSS.
```

Reply:
@hellofromtonya Addressed this as part of follow up https://github.com/WordPress/wordpress-develop/pull/3977/commits/2cf1b08be2f735f508c3e44c24154fc692cf6d75.